utils/osdetail/binmeta.go

```diff
@@ -67,6 +68,11 @@ func GenerateBinaryNameFromPath(path string) string {
 func cleanFileDescription(fileDescr string) string {
 	fields := strings.Fields(fileDescr)
 
+	// Clean out and `"` and `'`.
+	for i := range fields {
+		fields[i] = cleanName.ReplaceAllString(fields[i], "")
+	}
```

Comment:
Using [`strings.Replacer`](https://golang.org/pkg/strings/#Replacer) is probably more efficient than a regex search.

Reply:
Nice! Somehow was not aware of that.